presto-main/src/main/java/com/facebook/presto/sql/analyzer/SemanticErrorCode.java

```diff
@@ -55,6 +55,8 @@
 
     MISMATCHED_SET_COLUMN_TYPES,
 
+    INCONSISTENT_FIELD_COUNT,
```

Comment:
No need for a new error code. This is just a type mismatch error, since it's essentially an issue with rows in the VALUES having different types.